server/src/parser/parser.ts

```diff
@@ -242,9 +249,10 @@ export class Parser {
   }
 
   // parse function declaration
-  private declareFunction(scope?: Decl.Scope): INodeResult<Decl.Func> {
+  private declareFunction(scope?: Decl.Scope, typeHint?: Maybe<ITypeHint>): INodeResult<Decl.Func> {
```

Comment:
It doesn't appear anything is passed for `typeHint` here or in the other `declare` functions below. I think it would be good to remove them for now if this was temporary code.